### DISCUSSION TITLE
Error password reset MySQL

### DISCUSSION THREAD

**Discussion by jordon31** [leokhoa/laragon#449] [General]
![Screenshot 2023-03-21 211758](https://user-images.githubusercontent.com/94505651/226730897-6f772c51-04ac-430a-8f8b-c059826c3d7d.png)

How to fix?

**Comment by Pen-y-Fan** [comment]
Try this [SO How to reset the root password in MySQL 8.0.11?](https://stackoverflow.com/questions/50691977/how-to-reset-the-root-password-in-mysql-8-0-11), the `mysql_native_password` is the probable fix.

Let us know how you get on.

**Reply by jordon31** [reply]
solved thanks
